packages/rich-text/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.3 (Unreleased)
+
+### Internal
+
+- `unstableToDom` will no longer create a new DOM document on each call; a single document will be shared across each invocation.
```

Comment:
I thought we were not documenting unstable APIs?

Reply:
I hesitated, but ultimately decided that it's good to include _something_ in the CHANGELOG for any pull request.

I wouldn't so much call this out as documenting the unstable API, since a CHANGELOG is not documentation. The "Internal" heading was meant to indicate that it's more a refactoring / housekeeping than anything else.

Reply:
Maybe I can revise this a bit to avoid specifically mentioning `unstableToDom`, while keeping the essence of the note otherwise intact.